src/pages/post-comments/PostComments.jsx

```diff
@@ -0,0 +1,75 @@
+import { useState } from "react";
+import { useDispatch, useSelector } from "react-redux";
+import { useParams } from "react-router-dom";
+import { CommentCard, LeftSideBar, Post, RightSideBar } from "../../components";
+import { postComment } from "../../redux/reducers/postsSlice";
+import { InfinitySpin as Loader } from "react-loader-spinner";
+import "./postcomments.css";
+import { toast } from "react-toastify";
+
+export function PostComments() {
+  const dispatch = useDispatch();
+  const { postId } = useParams();
+  const allPosts = useSelector((state) => state.posts.posts);
+  const currentPost = allPosts.find((item) => item._id === postId);
+  const loading = useSelector((state) => state.posts.loading);
```

Comment:
suggestion: you can destructure `allPosts` and `loading` directly from `state.posts`